### DISCUSSION TITLE
FXGL 3D Game Ideas

### DISCUSSION THREAD

**Discussion by ImperaEtConquer** [AlmasB/FXGL#1016] [Ideas]
Hello everyone! I would like to discuss the 3D in FXGL. It is quite obvious that the capabilities are still pretty much at a very early stage. However, after the 3D API premiere on youtube, I (and I think many others too) got really hyped to try it out.

Therefore, I would like to discuss what small game is possible to develop right now?
I know that the PS1 visuals trend has been around for a while. Here's the example:
https://jribaux.itch.io/ps1-visuals-for-unity

The whole point in them is to make the game as low-poly and retro as possible. Players love this and feel nostalgic. It seems to me that even now FXGL may be a good choice for implementing such a game (for example, ps1 horror).

What do you think about it? Perhaps it is possible to use the FXGL to achieve this effect that PS1 games have? What other alternative 3D game ideas can there be other than what I suggested?

**Comment by AlmasB** [comment]
Good idea! I think there will be quite a bit of interest from the community to be involved with a 3D game (demo). So far, I've extracted the FPS camera development sample into a [separate repo](https://github.com/AlmasB/WolFX). This could potentially grow into something a bit bigger than just a sample.

**Reply by ImperaEtConquer** [reply]
Hey, @AlmasB !

Do you have any updates by chance regarding the PS1 visuals in the FXGL?

Do you think it is going to be possible to do something similar in JavaFX? Like can we add these jitters, texture stretches ditherings and etc so the 3DCamera could render this low-poly retro visual? Really want to try it out.

**Reply by AlmasB** [reply]
Hia, I'm not sure about PS1 visuals. Texture stretching seems feasible, you can try adding a Cuboid with a Texture that is low quality to give a similar look. Achieving authentic PS1 look, however, may not be as simple -- I haven't had a chance to think about this.
